src/components/Footer/Footer.js

```diff
@@ -0,0 +1,115 @@
+import {
+    Grid,
+    Container,
+    Button,
+    TextField,
+    Box,
+    Typography,
+    AppBar,
+    Toolbar,
+  } from "@mui/material";
+  
+  function Footer() {
```

Comment:
Please create index.js file in footer folder to export your component as we discussed in our meetings.